sonarqube/resource_sonarqube_portfolio.go

```diff
@@ -147,8 +146,8 @@ func resourceSonarqubePortfolio() *schema.Resource {
 }
 
 func checkPortfolioSupport(conf *ProviderConfiguration) error {
-	if strings.ToLower(conf.sonarQubeEdition) != "enterprise" {
-		return fmt.Errorf("portfolios are only supported in the Enterprise edition of SonarQube. You are using: SonarQube %s version %s", conf.sonarQubeEdition, conf.sonarQubeVersion)
+	if strings.ToLower(conf.sonarQubeEdition) == "community" {
```

Comment:
there is also a developer edition that does not support this. Can you change this condition to be != enterprise or data center

Reply:
yes, will do it now.